File: Karan_24b3003/Week 1/two_knights.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    for (int k = 1; k <= n; k++) {
        long long total = (long long)k * k * (k * k - 1) / 2;
        long long attack = 0;
        if (k > 2) 
            attack = 4LL * (k - 1) * (k - 2);
        cout << total - attack << "\n";
    }
    return 0;
}

File: Akshit_22b2531/week1/two_knights.py
n = int(input())
for k in range(1, n+1):
    total = k * k * (k * k - 1) // 2  # total ways to place two knights
    if k > 2:
        attacks = 4 * (k - 1) * (k - 2)  # ways two knights attack each other
    else:
        attacks = 0
    print(total - attacks)